pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -176,6 +181,19 @@ func (pl *VolumeRestrictions) PreFilter(ctx context.Context, cycleState *framewo
 	if err != nil {
 		return nil, framework.AsStatus(err)
 	}
+
+	needCheck := false
```

Comment:
Suddenly came up with a scenario, if pod with no volumes, and we disable the enableReadWriteOncePod, we should return Skip right? Not nil as success. Also we missed the corresponding test.

Reply:
Nice catch! Added one.